WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java

```diff
@@ -230,7 +230,7 @@ public void onErrorResponse(VolleyError error) {
                 mUrlSkipList.add(mRequestedURL);
             }
 
-            if (!mUrl.equals(mRequestedURL)) {
+            if (mUrl != null && !mUrl.equals(mRequestedURL)) {
```

Comment:
Should this be `if (mUrl == null || !mUrl.equals(mRequestedURL))`? Because if `mUrl` is null it likely means the request is no longer valid.